src/main/java/org/bihealth/mi/easybus/implementations/email/ConnectionIMAPSettings.java

```diff
@@ -123,26 +123,61 @@ public static void checkPort(int object) {
     private boolean                       ssltlsIMAP           = true;
     /** Use ssl/tls (=true) or starttls (=false) for SMTP connection */
     private boolean                       ssltlsSMTP           = true;
-    /** Password accessor */
-    private PasswordProvider              provider;
-
-    /**
-     * Creates a new instance
+    /** Password accessor for receiving */
+    private PasswordProvider              imapProvider;
+    /** E-mail address sending */
+    private String                        smptEmailAddress;
+    /** Password accessor for sending */
+    private PasswordProvider              smtpProvider;
+    /** IMAP Password */
+    private transient String              smtpPassword;
+    /** Logon name for IMAP */
```

Comment:
Logon -> User (will stop now to comment this. Should ideally be fixed by searching through the whole code.